src/components/LaunchFilter/LaunchFilter.scss

```diff
@@ -0,0 +1,9 @@
+
+.launch-filter-list {
+  position: fixed;
+  z-index: 2;
+  top:  6rem;
+  left: -2rem;
```

Comment:
why -2?

Reply:
z-index of slider - 1, i think filter should be placed above the slider.

Reply:
line 6 deleted
removed styles in Router component
added styles to Home component  with two containers